src/lib/components/blog/labels.svelte

```diff
@@ -34,15 +35,18 @@
     font-size: 1em;
 
     a {
-      background-color: var(--acm-light);
-      border-radius: 5px;
-      padding: 0.3em;
+      background-color: #eeeeee;
+      color: #212121;
+      border-radius: var(--size-sm);
+      border: 2px solid #e0e0e0;
```

Comment:
same reasons as above, maybe experiment if mixing acm-light and acm-canvas would produce a similar effect?
/

Reply:
You got it.